user_deletion/management/commands/delete_users.py

```diff
@@ -1,18 +1,22 @@
+from django.apps import apps
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.contrib.sites.models import Site
 from django.core.management.base import BaseCommand
 from django.utils import translation
 
-from ...notifications import AccountDeletedNotification
-
 User = get_user_model()
+user_deletion_config = apps.get_app_config('user_deletion')
```

Comment:
This is a bit of a redundant name I think. I'd probably have just called it `config`.